index.php

```diff
@@ -4,9 +4,81 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>Totally Pawesome</title>
+    <link rel="stylesheet" type="text/css" href="src/assets/css/vendor/bootstrap.min.css">
```

Comment:
How is this link to bootstrap different from the php autoload in line 1?

Reply:
Bootstrap is now to be given a relative path; the vendor file is for autoload / namespacing only.